src/Mono.Android/Test/Xamarin.Android.Net/AndroidClientHandlerTests.cs

```diff
@@ -79,7 +79,6 @@ public void Properties_Defaults ()
 			Assert.AreEqual (4096, h.CookieContainer.MaxCookieSize, "#3b");
 			Assert.AreEqual (null, h.Credentials, "#4");
 			Assert.AreEqual (50, h.MaxAutomaticRedirections, "#5");
-			Assert.AreEqual (int.MaxValue, h.MaxRequestContentBufferSize, "#6");
```

Comment:
@baulig Could you please shed some more light on why we return 0 for `MaxRequestContentBufferSize` property? I read the explanation in your [code](https://github.com/mono/mono/commit/2f411750acfd2132f9b4970ef8cf11d9a0f62df9#diff-a94223e905e684e2f789149da44fafd9R16), but didn't find any documentation about 0 meaning no limit. It only talks about the default value being 2 gigabytes, which is not the case now.

Should we really drop that check here on line 82?